### DISCUSSION TITLE
Need 2D / 3D Merged Editor

### DISCUSSION THREAD

**Discussion by jzq740176597** [godotengine/godot-proposals#6596] [Editor]
### Describe the project you are working on

2.5D platformer

### Describe the problem or limitation you are having in your project

**Single node** both has 2d-stuff <sprite2D> & 3D Node / Body / Collision. I just only see one aspect 2d or 3D stuff exclusive.
But is a single node NEED to sync work toggle as a whole.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

I Need See one node / scene 's 2d / 3d stuff in one view **simultaneously**

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

DO like unity's scene view, view 2d & 3D together!

### If this enhancement will not be used often, can it be worked around with a few lines of script?

no

### Is there a reason why this should be core and not an add-on in the asset library?

core

**Comment by AThousandShips** [comment]
Can you provide a screenshot of how this look, I can't find anything by searching for this

**Comment by Calinou** [comment]
- This is already being tracked in https://github.com/godotengine/godot-proposals/issues/2635.
